lib/fathead/caniuse/redirects.txt

```diff
@@ -0,0 +1,217 @@
+::selection element, ::selection css pseudo element
+selection element, ::selection css pseudo element
+selection pseudo elements, ::selection css pseudo element
+css selection element, ::selection css pseudo element
+ambient lights, ambient light
+ambient lights api, ambient light api
+ambient-lights, ambient-light
```

Comment:
@anjanjr ah thanks for clarifying -- we should fix that. There's no need create duplicate articles with different titles -- that's what redirects are for!

the mult-word titles should redirect to the single-word title with dashes.

We should adjust the parser so that it creates an Article for the dash-separated title, and a redirect for the space-separate to the dash separated.

Would you be interested in doing that?

This would also help to simplify and deduplicate the redirects.txt file. For example we'd be able to remove `ambient lights, ambient light`, and just keep:

```
ambient-lights, ambient-light
ambient lights, ambient light
```